neurom/fst/_mm.py

```diff
@@ -101,11 +101,6 @@ def map_sections(fun, nrn, neurite_type=NeuriteType.all):
                 for s in i_chain2(nrn.neurites, tree_filter=is_type(neurite_type)))
 
 
-def section_lengths(nrn, neurite_type=NeuriteType.all):
-    '''section lengths'''
-    return map_sections(mm.path_distance, nrn, neurite_type)
```

Comment:
Give me a while to confirm some definitions, because this could be a misunderstanding of the term path_distance in general...